Sources/Rendering/Core/ImageCPRMapper/example/index.js

```diff
@@ -29,6 +29,7 @@ import widgetBehavior from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget
 import controlPanel from './controller.html';
 import aortaJSON from './aorta_centerline.json';
 import spineJSON from './spine_centerline.json';
+import { ProjectionMode } from '../Constants';
```

Comment:
`import { ProjectionMode } from 'vtk.js/Sources/Rendering/Core/ImageCPRMapper/Constants';`

Reply:
Thank you for the review!
Is there a convention to always use absolute import?
I prefer using relative imports when the relationship between the two files is not likely to change in case of a refactor
For example:
```
|-- Utils/
|   |-- index.js
|-- Classes/
|   |-- MyClass/
|   |   |-- index.js
```
Here, from `Classes/MyClass/index.js`, I import `Utils/index.js` instead of `../../Utils/index.js` because the position of the `Utils` folder is unlikely to change, but `MyClass` could be for example renamed or moved

What do you think?

Reply:
Displaying relative will not help developers, the code of the examples is displayed on the [website ](https://kitware.github.io/vtk-js/examples/ImageCPRMapper.html) as a "starter" using the recommended version of vtk.js which is ESM, we can do relative import but will not to add another step to rewrite the import into ESM version import

*.js -> UMD version vtk.js/Sources
example/*.js -> ESM version @kitware/vtk.js
*.d.ts -> Relative import [rewritten ](https://github.com/Kitware/vtk-js/blob/master/rollup.config.js#L180-L207) into ESM at build

@floryst can give more insights

Reply:
Oh ok yes it makes a lot of sense for the example, I didn't think about it, thanks!